<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtOverlayControlsWidget</class>
 <widget class="QWidget" name="QtOverlayControlsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>166</width>
    <height>63</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Overlay</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item row="0" column="0" colspan="2">
    <widget class="QPushButton" name="LoadOverlayButton">
     <property name="text">
      <string>Load Overlay Image</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QSlider" name="OverlayOpacitySlider">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QDoubleSpinBox" name="OverlayOpacitySpinBox">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="maximum">
      <double>1.000000000000000</double>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
